From f78c48edda71593a5fa735a01dd178d5196145c0 Mon Sep 17 00:00:00 2001
From: Alexander Shiyan <shc_work@mail.ru>
Date: Sat, 30 Nov 2013 10:18:01 +0400
Subject: [PATCH 113/184] ARM: dts: imx27-phytec-phycore-som: Add on-flash BBT
 support

This patch adds support for On Flash Bad Block Table for PCM-038
onboard NAND.

Signed-off-by: Alexander Shiyan <shc_work@mail.ru>
Signed-off-by: Shawn Guo <shawn.guo@linaro.org>
---
 arch/arm/boot/dts/imx27-phytec-phycore-som.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx27-phytec-phycore-som.dts b/arch/arm/boot/dts/imx27-phytec-phycore-som.dts
index 8771928..beec525 100644
--- a/arch/arm/boot/dts/imx27-phytec-phycore-som.dts
+++ b/arch/arm/boot/dts/imx27-phytec-phycore-som.dts
@@ -201,6 +201,7 @@
 &nfc {
 	nand-bus-width = <8>;
 	nand-ecc-mode = "hw";
+	nand-on-flash-bbt;
 	status = "okay";
 };
 
-- 
1.8.5.3

